### DISCUSSION TITLE
CloudQuery Source Plugin?

### DISCUSSION THREAD

**Discussion by yevgenypats** [plausible/analytics#2611] [Anything Else]
Hi Team, hopefully this is right place to ask, if not, I'd appreciate if you can direct me.

I'm the founder of [cloudquery.io](https://www.cloudquery.io/), a high performance open source ELT framework. 

Our users are interested in a Plausible plugin, but as we cannot maintain all the plugins ourselves, I was curious if this would be an interesting collaboration, where we would help implement an initial [source plugin](https://www.cloudquery.io/docs/plugins/sources/overview), and you will help maintain it.

This will give your users the ability to sync Plausible data to any of their datalakes/data-warehouses/databases easily using any of the growing list of CQ [destination plugins](https://www.cloudquery.io/docs/plugins/destinations/overview).

Best,
Yevgeny

**Comment by metmarkosaric** [comment]
thanks for reaching out @yevgenypats! the plugin sounds exciting!

we're growing really fast and have limited development resources so all our focus is on the current gaps in our product itself (custom dimensions for pageviews, comparisons, 2FA, funnels...). this is why we cannot help with any third-party integration development or maintenance. sorry about that!

if you do decide to go ahead with this plugin, please do share it when it's ready. we're happy to promote it in our docs, github and tell people via email/social when they ask us about integrations with the different data warehouses etc. we do something similar with a third-party developed Data Studio connector and have heard very positive feedback from that developer for whom the Plausible connector is the fastest growing product. 

thanks for understanding!

**Reply by yevgenypats** [reply]
Hi @metmarkosaric Thanks for the quick response. This makes sense. I think we will develop such an integration then if you will be able to promote it in your docs this would be amazing. If the integration will gain enough traction we will be open to transferring the repo to your organization if you would like to get direct feedback from your users but we are ok with maintaining it for starters as long as you can help us promote it!

**Reply by metmarkosaric** [reply]
sounds like a plan @yevgenypats! do keep us posted on this and we'll add it to the relevant places to promote it.

**Reply by yevgenypats** [reply]
Hi @metmarkosaric ! Check out our new plausible source plugin with example to sync the timeseries api to a data warehouse / data lake - https://www.cloudquery.io/docs/plugins/sources/plausible/overview .

Would love if you can help us spread the word! Specifically I think the following would be helpful but whatever you think is best as you know your audience :) :
- https://plausible.io/docs/stats-api -  I think we solve a use-case here were users can export the whole timeseries history by running a periodic CloudQuery cron (as this is not available in the UI and exporting everything is not always possible if it's much info or can be slow).
- Twitter,Linkedin - we will be happy to retweet so let me know when you share this 🚀 
- Any other channel that you think is good 🙏 

I'll be happy to provide more context or any additional examples, blurb that is needed to make the most out of it!

**Reply by metmarkosaric** [reply]
this is great @yevgenypats, thank you! i didn't expect it to be done so fast! 

we've started including links to this already:

- there are some in our integration list here: https://plausible.io/docs/integration-guides
- we're posting it in the relevant github threads such as this one: https://github.com/plausible/analytics/discussions/1777
- we'll look through our email archieves and email it to those who asked for connections to datawarehouses
- plus will be sending this in the future when someone asks about bigquery, snowflake, s3 etc

**Reply by yevgenypats** [reply]
Thank you, this is great 🙏

**Reply by yevgenypats** [reply]
@metmarkosaric Also just announced here! Feel free to like/RT :)

https://www.linkedin.com/feed/update/urn:li:activity:7025732244879065088
https://twitter.com/cloudqueryio/status/1619966533299744769

**Comment by arthurburle** [comment]
Hi guys,
The link to the plausible cloudquery plugin seems not to be available anymore, do you know what happened?
I am really interested on switching from google analytics to plausible but I need to keep sending my data to bigquery. Do you have other ideas for doing so?

**Comment by arthurburle** [comment]
Hi, @yevgenypats and @metmarkosaric,
Sorry for tagging you guys here. Can you check my last comment please?
